packages/reports/addon/templates/components/metric-selector.hbs

```diff
@@ -13,54 +13,56 @@
     as | metric |
   }}
-    <label class="grouped-list__item-label">
-      <input class="checkbox-selector__checkbox" type="checkbox" onchange={{action "metricClicked" metric}} checked={{get metricsChecked metric.name}}>
+    <div class="grouped-list__item-container {{if (and (get metricsChecked metric.name) enableRequestPreview) "grouped-list__item-container--selected"}}">
+      <NaviIcon
+        @icon={{if (and (get metricsChecked metric.name) (not enableRequestPreview)) "minus-circle" "plus-circle"}}
+        class="grouped-list__add-icon {{if (not (or enableRequestPreview (get metricsChecked metric.name))) "grouped-list__add-icon--deselected"}}"
+        onclick={{action "metricClicked" metric}} />
       {{metric.longName}}
-    </label>
 
-    {{#navi-icon "question-circle-o" class="grouped-list__item-info"}}
-      {{#ember-tooltip side="right" popperContainer="body" effect="none"}}
-        {{#if metric.extended.isPending}}
-          {{navi-loading-message}}
-        {{else}}
-          {{or metric.extended.content.description "No description available"}}
-        {{/if}}
-      {{/ember-tooltip}}
-    {{/navi-icon}}
+      {{#navi-icon "question-circle-o" class="grouped-list__item-info"}}
+        {{#ember-tooltip side="right" popperContainer="body" effect="none"}}
+          {{#if metric.extended.isPending}}
+            {{navi-loading-message}}
+          {{else}}
+            {{or metric.extended.content.description "No description available"}}
+          {{/if}}
+        {{/ember-tooltip}}
+      {{/navi-icon}}
 
-    <div class="metric-selector__icon-set {{if (not (can-having metric)) "metric-selector__icon-set--no-filter"}}">
-      {{#if (get metric "hasParameters")}}
-        {{metric-config
-          metric=metric
-          request=request
-          onAddParameterizedMetric=(update-report-action "ADD_METRIC_WITH_PARAM")
-          onRemoveParameterizedMetric=(update-report-action "REMOVE_METRIC_WITH_PARAM")
-          onToggleParameterizedMetricFilter=(update-report-action "TOGGLE_PARAMETERIZED_METRIC_FILTER")
-        }}
-      {{/if}}
-      {{#if (can-having metric)}}
-        {{#navi-icon
-          "filter"
-          class=(concat (if (get metricsFiltered metric.name) "checkbox-selector__filter--active ") "checkbox-selector__filter metric-selector__filter")
-          click=(action onToggleMetricFilter metric)
-        }}
-          {{#ember-tooltip side="right" popperContainer="body" effect="none"}}
-            {{#if (has-parameters metric)}}
-              {{#if (has-unfiltered-parameters metric request)}}
-                Add next filter
-              {{else}}
-                Remove all
-              {{/if}}
-            {{else}}
-              {{#if (get metricsFiltered metric.name)}}
-                Remove Filter
+      <div class="grouped-list__icon-set {{if (not (can-having metric)) "metric-selector__icon-set--no-filter"}}">
+        {{#if (get metric "hasParameters")}}
+          {{metric-config
```

Comment:
Can we do `<MetricConfig/>`

Reply:
yeah, didn't want to change all that because there were enough changes for 1 PR